src/content/blog/you-probably-don't-need-eslint-config-prettier-or-eslint-plugin-prettier/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+description: "Explaining two popular Prettier integrations for ESLint, and why I avoid both of them."
+image:
+  alt: Josh smiling on stage and giving a presentation showing the "Type Checking React (v2)" from the linked slides.
+  src: react-miami-me-speaking.jpg
+pubDate: 2023-01-14
```

Comment:
I'm thinking I'll want to publish this post _after_ #141. If I do this one before, some folks are probably going to low-effort reply with mentions of Rust linters.